rockcraft/models/project.py

```diff
@@ -87,9 +87,24 @@
     ),
 ]
 
-BaseT = Literal["bare", "ubuntu@20.04", "ubuntu@22.04", "ubuntu@24.04"]
+BaseT = Literal[
+    "bare",
+    "ubuntu@20.04",
+    "ubuntu@22.04",
+    "ubuntu@24.04",
+    "ubuntu@25.04",
+    "ubuntu@25.10",
```

Comment:
I'm blanking on where we implement the behavior of devel bases: specifically here, if `base: ubuntu@25.10` then `build-base` **must** be `devel`

Reply:
do we still support that via craft-application? [reference](https://github.com/canonical/rockcraft/commit/97e0481790e018d7eee390d65f371c8ad5cebb59)

Reply:
My answer to you is the worst of all possible answers!

1. Yes, we [do still support it](https://github.com/canonical/craft-application/blob/c5fdcf0521a7c026cace5883bd7590fc841536ed/craft_application/models/project.py#L273-L303) in craft-application
2. It's [currently a no-op 😬](https://github.com/canonical/craft-application/blob/c5fdcf0521a7c026cace5883bd7590fc841536ed/craft_application/models/project.py#L58-L64)
3. Now that we're getting up-to-date distro support data files, we should really rewrite it to use that. (`if today < base.release_date...`)

- [Short term](https://github.com/canonical/craft-application/issues/880)
- [Long term](https://github.com/canonical/craft-application/issues/881)

Reply:
Hmm... that's a good point, but it makes things complicated since different apps may want to do that at different times. I've put your comment in the issue so we can discuss further.